### PR TITLE
Build fix for apple operating system

### PR DESCRIPTION
**Pull request by p-vital**

### CODE REVIEW

**Comment by giampaolo** [comment]
I don't know C++ so I cannot review this; just wanted to say it's great to have someone who's able to do this, otherwise I guess we would have been pretty screwed. :D

**Comment by p-vital** [comment]
I just wanted someone to confirm that the fix works.

**Comment by rabialam** [comment]
@mthomas or @colorstain, wanna try a build with this patch to see if it works?

File: setup.py [hunk 1]
```diff
@@ -20,6 +20,12 @@ def get_swigdir():
 
 from distutils.core import setup, Extension
 
+import platform
+if platform.system() == 'Darwin':
```
Comment (giampaolo) [inline setup.py]: To be extra sure I would rather use `sys.platform.startswith("darwin")` - I've been doing that in psutil for years and never received any complaint.

Comment (p-vital) [inline setup.py]: That's cool with me - please feel free to make this change.

**Pull request merged**